example-app/SantokuApp/src/screens/demo/useSpecAndSouceCodeLink.tsx

```diff
@@ -0,0 +1,33 @@
+import React, {useCallback} from 'react';
+import {Linking, StyleSheet, View} from 'react-native';
+import {Button} from 'react-native-elements';
+
+import {AppConfig} from '../../framework';
+
+const openUrl = async (url: string) => {
+  await Linking.openURL(url);
+};
+
+export const useSpecAndSourceCodeLink = (feature: string) => {
```

Comment:
Why are you using custom hooks? Wouldn't a functional component suffice?

Reply:
It doesn't mean much.
I even thought it might be a good idea to return components from the use case.

However, it is not consistent with other components, so I changed it to functional component
1473f66942fd295e6f2a692b36744726b8142945